src/main/resources/templates/login.html

```diff
@@ -40,13 +40,45 @@
 				<input type="password" class="login_text input_text self-format" ng-model="login.credentials.password"
 					name="password">
 			</div>
-			<div id="submit_button"  class="common_industry industry_font"> 
+			<div id="submit_button"  class="common_industry industry_font">
 				<button class="btn common_div industry_font" type="submit" translate="SIGN_IN"></button>
 			</div>
+			<div id="forgot_password" class="common_industry login_text">
+				<a translate="FORGOT_PASSWORD" data-toggle="modal" data-target="#forgotPasswordModal"></a>
+			</div>
 		</form>
 	</div>
 	<div ng-include="'common/pages/footer.html'"></div>
-	
+	<div class="modal fade industry_font" id="forgotPasswordModal" tabindex="-1" role="dialog" aria-hidden="true" data-backdrop="static">
+		<div class="modal-dialog">
+			<div class="modal-content">
+				<div class="modal-header">
+					<button type="button" class="close font_color1 green_text" data-dismiss="modal">×</button>
+					<h4 class="modal-title"><span translate="USER_INFORMATION"></span></h4>
+				</div>
+				<div class="modal-body">
+					<table class="password-table">
+						<tr class="table-header">
+							<td><span translate="USER_NAME"></span></td>
+							<td><span translate="USER_PASSWORD"></span></td>
+						</tr>
+						<tr>
+							<td>purchaser</td>
+							<td>password</td>
```

Comment:
It's possible that user could modify the username/password here.
So instead of using hard-coded string, we should read the first approver/purchaser's information here.